src/wiki/development/build-instructions.md

```diff
@@ -330,17 +330,18 @@ This is where the final `PrismLauncher.app` will be constructed when you run `ma
 mkdir build
 cd build
 cmake \
+ -G Ninja \
  -DCMAKE_BUILD_TYPE=Release \
  -DCMAKE_INSTALL_PREFIX="dist" \
  -DCMAKE_PREFIX_PATH="/path/to/Qt/" \
  -DQt6_DIR="/path/to/Qt/" \
- -DCMAKE_OSX_DEPLOYMENT_TARGET=10.14 \
+ -DCMAKE_OSX_DEPLOYMENT_TARGET=11 \
  -DLauncher_BUILD_PLATFORM=macOS \
- -DENABLE_LTO=ON # if you want to enable LTO/IPO:
+ -DENABLE_LTO=ON \ # if you want to enable LTO/IPO:
 #-DCMAKE_OSX_ARCHITECTURES="x86_64;arm64" # to build a universal binary (not recommended for development)
 #-DLauncher_QT_VERSION_MAJOR=5 \ # if you want to use Qt 5
  ..
-make install
+cmake --build .
```

Comment:
since we're at it, shouldn't we also add the install part?

https://github.com/PrismLauncher/PrismLauncher/blob/850e48ec679fb62426fb03ace582a3cf147dfc43/.github/workflows/build.yml#L371

Reply:
sure

Reply:
well, that depends if the user wants to install it in their applications folder